### ISSUE TITLE
spotlight tour

### ISSUE THREAD

**Issue by Adarsh1993** [stackbuilders/react-native-spotlight-tour#52]
Thanks for great package .

How to hints show in first time  when app is start .

second time disable . This is possible with spotlight tour.

Thank you

**Comment by JoseLion** [comment]
Hi @Adarsh1993, I'm glad you're finding this package useful 🙂 

It's possible to start the tour as soon as the app begins using a ref and a `useEffect(..)`. We had a suggestion about making this easier in #50, and we'll be implementing a prop for that soon. You can also find a complete example of it in that issue.

There's no built-in feature to start the tour only the first time, but you can add that logic yourself in the `useEffect` mentioned above. You can use a package like [async-storage](https://github.com/react-native-async-storage/async-storage) to store a per-device flag. Then you can start the tour base on this flag and update the flag once the tour has finished.

This could be a great feature for us to implement since most people might be using the tour this way, and we could reduce the burden of managing the stored flags. However, we'd have to evaluate the details because that'd mean adding a dependency on [async-storage](https://github.com/react-native-async-storage/async-storage) or a similar package.

I'll leave this ticket open to keep you posted on the topic. Thanks again! Your feedback helps us make this package better and better 🙂

**Comment by mohamedabkal** [comment]
@JoseLion Yes please this feature is very important. If you can add it it would amazing! 💪

**Comment by jakequade-hsm** [comment]
This is on the developer to implement, not on the library. Use something like a `useEffect` in conjunction with `AsyncStorage` to set a `hasSeenTour` value.
